index.html

```diff
@@ -0,0 +1,120 @@
+<DOCTYPE! html>
+<html>  
+  <head>
+    <!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="styles.css">    
+  </head>
+  <body>
+    <!-- placeholders for everything held in navbar -->
+    <nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-xs-2 navtitle"><p>Project Name</p></div>
+          <div class="col-xs-2 col-xs-offset-4 emailtime">
+            <div class="form-group">
+              <label for="exampleInputEmail1"></label>
+              <input type="email" class="form-control" id="exampleInputEmail1" placeholder="Enter email">
+            </div>
+          </div>
+          <div class="col-xs-2 password">  
+            <div class="form-group">
+             <label for="exampleInputPassword1"></label>
+             <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+            </div>
+          </div>  
+          <div class="col-xs-1 signin">
+           <button type="button" class="btn btn-primary" data-toggle="button" aria- pressed="false"  autocomplete="off">
+            Sign In
+           </button>
+          </div>
+          
+        </div>
+      </div>
+    </nav>
+   <div class="container">

```

Comment:
Using "xs" column sizes throughout, so the elements don't collapse properly at small sizes.